policy-report/image-vuln-adapter/pkg/apis/wgpolicyk8s.io/v1alpha1/clusterpolicyreport_types.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
As discussed, we should extract this to a common package and reuse it. We can the remove it from here.

Reply:
yeah